app/views/manage_assignments/courses/_attachments.html.erb

```diff
@@ -1,12 +1,10 @@
-<table class="assignment-attachments is-hidden">
-  <tbody>
-    <% attachments.each do |attachment| %>
-      <tr>
-        <td><%= attachment.file_file_name %></td>
-        <td><%= link_to t(".delete-link"),
-              manage_assignments_attachment_path(attachment),
-              method: :delete %></td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
+<ul class="assignment-attachments is-hidden attachments">
```

Comment:
@alexandra-aas Interesting point. I usually just start tacking things on in no particular order. My thought is that even if I do order them by specificity the next person to touch the code will have to deduce that. It's not obvious solely from the markup.